.husky/pre-commit

```diff
@@ -4,4 +4,4 @@
 pnpm lint
 pnpm format:check
 pnpm types:check
-pnpm locale:verify
\ No newline at end of file
+pnpm locale:generate
```

Comment:
Don't think this adds any value? Since all.json is not commited. It would simple fail if someone changed locale data, but forgot to generate the new locale. But that will have no impact on the release.